wgsl/index.bs

```diff
@@ -10998,11 +11000,11 @@ struct __modf_result_vecN_f16 {
   </thead>
   <tr algorithm="determinant">
     <td>|T| is AbstractFloat, f32, or f16
-    <td class="nowrap">`determinant(`|e|`:` mat|C|x|C|<|T|> `) -> ` |T|
+    <td class="nowrap">`@const fn`<br>`determinant(`|e|`:` mat|C|x|C|<|T|> `) -> ` |T|
     <td>Returns the determinant of |e|.
   <tr algorithm="transpose">
     <td>|T| is AbstractFloat, f32, or f16
-    <td class="nowrap">`transpose(`|e|`:` mat|R|x|C|<|T|> `) -> ` mat|C|x|R|<|T|>
+    <td class="nowrap"`@const fn`<br>`transpose(`|e|`:` mat|R|x|C|<|T|> `) -> ` mat|C|x|R|<|T|>
```

Comment:
Have to terminate the `td` element before the `@const` text.
That's the reason for the validation failure.